github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -1627,7 +1627,7 @@ periodics:
       - name: KUBEVIRT_QUARANTINE
         value: "true"
       - name: TARGET
-        value: k8s-1.23-operator
```

Comment:
Shouldn't this stay?

Reply:
No, because the `KUBEVIRT_E2E_FOCUS` is set also, so we don't need it here.

Reply:
See the 1.22 config here as a reference: https://github.com/kubevirt/project-infra/blob/9e686b59210a187c67681fbe1fd25876969e0b30/github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml#L1332

Reply:
Thanks, I missed it.